README.md

```diff
@@ -41,12 +41,18 @@ Example usage:
     # TYPE kaiterra_particulate_matter gauge
     kaiterra_particulate_matter{microns="10"} 16
     kaiterra_particulate_matter{microns="2.5"} 15
+    # HELP kaiterra_rtvoc Total Volatile Organic Compounds (TVOC) in ppb
+    # TYPE kaiterra_rtvoc gauge
+    kaiterra_rtvoc 486
```

Comment:
Update with new names?

Reply:
Oh, wait, delete these lines I think, theres new lines below.

Reply:
Fixed it